packages/expo-camera/src/next/Camera.types.ts

```diff
@@ -269,32 +269,20 @@ export type BarcodeScanningResult = {
 
 export type ModernBarcodeScanningResult = Omit<BarcodeScanningResult, 'bounds'>;
 
-/**
- * @hidden
- */
-export type ConstantsType = {
-  Type: CameraType;
-  FlashMode: FlashMode;
-  VideoQuality: VideoQuality;
-  VideoStabilization: VideoStabilization;
-  VideoCodec: VideoCodec;
-};
-
 // @needsAudit
 export type CameraProps = ViewProps & {
   /**
    * Camera facing. Use one of `CameraType`. When `front`, use the front-facing camera.
    * When `back`, use the back-facing camera.
    * @default 'back'
    */
-  type?: CameraType;
+  facing?: CameraType;
   /**
-   * Camera flash mode. Use one of [`FlashMode.<value>`](#flashmode-1). When `FlashMode.on`, the flash on your device will
-   * turn on when taking a picture, when `FlashMode.off`, it won't. Setting to `FlashMode.auto` will fire flash if required,
-   * `FlashMode.torch` turns on flash during the preview.
+   * Camera flash mode. Use one of `FlashMode`. When `on`, the flash on your device will
```

Comment:
```suggestion
   * Camera flash mode. Use one of `FlashMode` values. When `on`, the flash on your device will
```

Is it one of the flash mode values to be used here? Is that assumption correct for the second sentence.

Reply:
Yes, that's correct